app/components/Chatbot/widgit.css

```diff
@@ -20,3 +20,16 @@
     width: 100%;
   }
 }
+
+.settings-container {
+  position: absolute;
+  bottom: var(--spacing-16);
+  left: var(--spacing-16);
+}
+
```

Comment:
Aw thanks. Feel free to do the minimum when it comes to CSS

Reply:
oh, I will :D